examples/todolist/tests/bindings/test_todolist.kts

```diff
@@ -0,0 +1,23 @@
+import uniffi.todolist.*
+
+val todo = TodoList()
+todo.addItem("Write strings support")
+
+assert(todo.getLast() == "Write strings support")
+
+todo.addItem("Write tests for strings support")
+
+assert(todo.getLast() == "Write tests for strings support")
+
+val entry = TodoEntry("Write bindings for strings as record members")
+
+todo.addEntry(entry)
+assert(todo.getLast() == "Write bindings for strings as record members")
+assert(todo.getLastEntry().text == "Write bindings for strings as record members")
+
+todo.addItem("Test Ünicode hàndling without an entry can't believe I didn't test this at first 🤣")
```

Comment:
I literally laughed out loud at the way you've incorporated an emoji into this test string, what was quite embarrassing given that I was checking my email on my phone in a public place 👍